lib/p2p/Pool.ts

```diff
@@ -121,22 +130,32 @@ class Pool extends EventEmitter {
 
   private handlePacket = (peer: Peer, packet: Packet) => {
     switch (packet.type) {
-      case PacketType.ORDER: {
+      case PacketType.ORDER:
         const order: PeerOrder = { ...packet.body, hostId: peer.id };
         this.emit('packet.order', order);
         break;
-      }
-      case PacketType.GETORDERS: {
+      case PacketType.GET_ORDERS:
         this.emit('packet.getOrders', peer);
         break;
-      }
-      case PacketType.ORDERS: {
-        const { orders } = packet.body;
+      case PacketType.ORDERS:
+        const { orders } = (packet as OrdersPacket).body;
         orders.forEach((order) => {
           this.emit('packet.order', { ...order, hostId: peer.id });
         });
         break;
-      }
+      case PacketType.GET_HOSTS:
+        peer.sendHosts(this.hosts.toArray());
```

Comment:
same issue as in https://github.com/ExchangeUnion/xud/issues/208 applies here as well. Need to follow the ping/pong example.